src/main/java/net/minecraftforge/common/IHorseArmor.java

```diff
@@ -0,0 +1,22 @@
+package net.minecraftforge.common;
+
+import net.minecraft.entity.passive.EntityHorse;
+import net.minecraft.entity.passive.HorseArmorType;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.World;
+
+public interface IHorseArmor 
+{
+    /** 
+     * @param stack - The armor stack
```

Comment:
Incorrect Javadoc formatting. After `@param`, there should be no dashes or any other kind of separator, because that's what the Javadoc compiler is for. Also, if the first phrase of the string is not a full sentence, then it is not capitalized, and if that first non-sentence phrase is the only phrase, it is not ended with a period. If there is another phrase, then there is always a period, and every phrase that is not the first should be a sentence. [Reference (`@param`)](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#@param)

This line should be:

    @param stack the armor stack